src/webview/MessageList.js

```diff
@@ -176,10 +180,11 @@ class MessageListInner extends React.Component<Props> {
     const contentHtml = messageListElementsForShownMessages
       .map(element => messageListElementHtml({ backgroundData, element, _ }))
       .join('');
-    const { auth, theme } = backgroundData;
```

Comment:
It looks like this is the one place where we were using the `theme` property on this `backgroundData` object. So, in either this same commit or a tiny followup commit, let's drop this no-longer-used property from there, too.

Reply:
Done. I added it as a small separate commit.